central/apitoken/service/service_impl.go

```diff
@@ -146,6 +147,7 @@ func (s *serviceImpl) ListAllowedTokenRoles(ctx context.Context, _ *v1.Empty) (*
 			result = append(result, role.GetRoleName())
 		}
 	}
+	sort.Strings(result)
```

Comment:
https://pkg.go.dev/sort@go1.21.9#Strings is preferred for perf reasons (plus in 1.22, `sort.Strings` just calls `slices.Sort`)

```suggestion
	slices.Sort(result)
```

Reply:
interesting, thank you for your comment. I see that sort.Strings(result) is a standard library function, while slices.Sort was also used in other scripts but as a custom defined function.  Such as the slices.Sort//Func sort for the missing tests, or the slices.SortFunc(oneOfFieldTypes, oneOfFieldTypeCmp). Thank you:)